extensions/wikia/ContentReview/services/ContentReviewStatusesService.class.php

```diff
@@ -313,7 +313,7 @@ private function createRevisionLink( \Title $title, $revisionId, $liveRevisionId
 	private function createRevisionTalkpageLink( \Title $title ) {
 		return \Html::element(
 			'a',
-			[ 'href' => $title->getTalkPage() ],
+			[ 'href' => $title->getTalkPage()->getLocalURL() ],
```

Comment:
Thanks for the fix. I just wanted to see rejection message more easily so that fix is the best!!!